package.json

```diff
@@ -23,17 +23,17 @@
   },
   "homepage": "https://github.com/digitalbazaar/bedrock-kms",
   "dependencies": {
-    "did-io": "^0.8.2",
-    "did-method-key": "^0.7.0",
-    "did-veres-one": "^12.1.1",
-    "jsonld-signatures": "^7.0.0",
+    "@digitalbazaar/ed25519-signature-2020": "^2.0.0",
+    "bedrock-did-io": "digitalbazaar/bedrock-did-io#initial",
```

Comment:
`bedrock-did-io` should be a peerDependency here, and a regular dependency in the test suite.